src/Exceptions/CommandExecutionException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+declare(strict_types=1);
+
+
+namespace PHPSu\Exceptions;
+
+class CommandExecutionException extends \Exception
```

Comment:
I don't like this a lot

Reply:
@Kanti addressing this in a different pr with a better php-api safe approach